packages/react-utilities/src/compose/README.md

```diff
@@ -74,120 +82,62 @@ const ToggleButton = React.forwardRef((props, ref) => {
   const state = useToggleButton(props, ref);
 
   // Inject classNames as needed.
-  state.className = css(
-    state.className,
-    styles.root,
-    state.checked && styles.checked
-  );
+  state.className = mergeClasses(state.className, styles.root, state.checked && styles.checked);
 
   return renderButton(state);
+});
 ```
 
 ### Details
 
-#### Creating mutable state with `mergeProps`
-
-In the previous example, `_.merge` was used to deep clone the props into a state object. Creating a single clone and
-using that to construct state simplifies hook development and usage; rather than trying to re-clone objects
-unnecessarily on every small mutation, hooks can assume operating against a draft state. This creates more self
-contained hooks, which can ensure they apply state updates correctly, avoiding accidents like stomping on existing event
-handlers by blind object assigning the results.
-
-However, deep merge overlooks many edge cases for component props:
-
-- Deep merging classnames should append them, not replace
-- Deep merging JSX, ref objects, or arrays should replace, not recurse/clone
-- Deep merging an object on a string should replace
-
-...which introduces the first utility: `mergeProps`. Merge props works like a deep merge, but takes care of classnames,
-JSX, arrays, and object edge cases.
-
-#### Supporting the `as` prop with `getSlots`
-
-Fluent UI components take a common `as` prop. This allows the root element to be rendered with something other than the
-default.
-
-To support the `as` prop, the render function might look like this:
-
-```jsx
-const renderButton = state => {
-  const root = state.as;
-
-  return <root {...state} />;
-};
-```
-
-Additionally, you will need to filter out native properties which apply to the root; otherwise you will end up mixing
-any unexpected props into the element. To do this, we have a `getNativeElementProps` helper, which can be used for this
-purpose:
-
-```jsx
-const renderButton = state => {
-  const root = state.as;
-  const rootProps = getNativeElementProps(root, state);
-
-  return <root {...rootProps} />;
-};
-```
-
-These steps have been abstracted in the `getSlots` helper:
-
-```jsx
-const renderButton = state => {
-  const { slots, slotProps } = getSlots(state);
-
-  return <slots.root {...slotProps.root} />;
-};
-```
```

Comment:
This was changed in RFC #19068. Removed completely as don't have good ideas how to replace it.